### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by englehardt**
I've had to add:

```
# localstack compatibility
more-itertools==5.0.0
zipp==0.6.0
```

due to localstack's outdated pinned dependencies

and we already pin `psutil` for the same reason. 

We should investigate what it would take to run localstack as a docker image to avoid these issues in the future, since the current installation approach we use [is deprecated](https://github.com/localstack/localstack/issues/2019#issuecomment-582581192).

### CODE REVIEW

**Comment by englehardt** [comment]
This needs to be rebased on #593. @vringar you can hold off on reviewing for now.

**Comment by englehardt** [comment]
This is now ready for review

File: automation/Extension/webext-instrumentation/package.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
     "@types/firefox-webext-browser": "^63.0.0",
```
Comment (vringar) [inline automation/Extension/webext-instrumentation/package.json:64]: Could we bump this to [latest version](https://www.npmjs.com/package/@types/firefox-webext-browser) 
```suggestion
    "@types/firefox-webext-browser": "^70.0.1",
```

Comment (englehardt) [reply automation/Extension/webext-instrumentation/package.json]: Done!

**Pull request merged**